evalml/pipelines/utils.py

```diff
@@ -144,46 +145,6 @@ def make_pipeline(X, y, estimator, problem_type, parameters=None, custom_hyperpa
     return base_class(complete_component_graph, parameters=parameters, custom_hyperparameters=custom_hyperparameters)
 
 
-def make_pipeline_from_components(component_instances, problem_type, custom_name=None, random_seed=0):
```

Comment:
I'm debating whether we should keep this around for another release and instead issue a deprecation warning in this pr.

I'm ok with deleting this util, I don't think it got a lot of traction from users but just noting that I think we use a different process for deleting functions/methods.

Reply:
Interesting point, but I'm okay deleting immediately, since I think the main use of this was internal alignment :p Plus, it's not too difficult to do what `make_pipeline_from_components` with our new pipelines so not too much lost there!